tests/system/robot/chromeTests.py

```diff
@@ -140,6 +140,125 @@ def test_mark_aria_details_FreeReviewCursor():
 	exercise_mark_aria_details()
 
 
+def test_mark_aria_details_role():
+	_chrome.prepareChrome(
+		"""
+		<div class="editor" contenteditable spellcheck="false" role="textbox" aria-multiline="true">
+			<p>
+				<span aria-details="endnote-details">doc-endnote,</span>
+				<span aria-details="footnote-details">doc-footnote,</span>
+				<span aria-details="comment-details">comment,</span>
+				<span aria-details="definition-details">definition,</span>
+				<span aria-details="unknown-details">form</span>
+			</p>
+		</div>
+		<div>
+			<p>
+				<div id="endnote-details" role="doc-endnote">details with role doc-endnote</div>
+				<div id="footnote-details" role="doc-footnote">details with role doc-footnote</div>
+				<div id="comment-details" role="comment">details with role comment</div>
+				<div id="definition-details">details with role definition</div>
+				<div id="unknown-details" role="form">details with role form</div>

```

Comment:
Why is this here? Because it is unsupported, please add a comment in the test data to clarify.

Reply:
Yes, to test an unsupported role

Reply:
This is documented in comments in `expectedSpeech` and `expectedBraille`.

I'm uncertain on the best way to add comments to the html multi-line string

Reply:
You could:
- add a html comment `<!-- a note here -->`
- Split the string and add a python comment in between

``` python
"""
some string contents
""" 
# the next line is here for xyz
"""
some interesting line
"""
```

I think the html comment is the better option, then it is in the test case when loaded by the browser.